orangecontrib/imageanalytics/import_images.py

```diff
@@ -171,7 +172,7 @@ def create_table(image_meta, categories=None, start_dir=None):
         # Image name (file basename without the extension)
         imagename_var = Orange.data.StringVariable.make("image name")
         # Full fs path
-        image_var = Orange.data.StringVariable.make("image")
+        image_var = Orange.data.StringVariable.make(ImgAttrName)
```

Comment:
I suggest an additional change to this line since `make` is not required anymore. 

```suggestion
        image_var = Orange.data.StringVariable(IMG_ATTR_NAME)
```